src/core/adaptors/forgetPassword/index.adaptors.ts

```diff
@@ -22,15 +20,20 @@ export const forgetPasswordAdaptor = async (email: string): Promise<AdaptorRes<S
 
 export const resetPasswordAdaptor = async (password: string): Promise<AdaptorRes<SuccessRes>> => {
   try {
-    //TODO: call resetPassword API
-    // TODO: map the result
-    const data = {
-      message: 'succeed',
-    };
-    return {
-      data,
-      error: null,
-    };
+    const res = await updatePassword({ password });
+    if (res.message === 'success') {
+      const data = {
+        message: 'succeed',
+      };
+      return {
```

Comment:
no need to handle error in `try` block

Reply:
I am not sure what other messages might get back from API. However, I changed it. We can handle more states later